tests/nematus.ini

```diff
@@ -87,7 +87,7 @@ rnn_cell="NematusGRU"
 ; This block just fills the arguments of the trainer __init__ method.
 class=trainers.cross_entropy_trainer.CrossEntropyTrainer
 decoders=[<decoder>]
-l2_weight=1.0e-8
+regularizers=[trainers.regularizers.L2]
```

Comment:
klidně bych to z těch testů vyhodil a v jednom jich otestoval víc

Reply:
Jo, muzeme to povyhazovat